### PR TITLE
feat: re-added related article component

### PR DESCRIPTION
**Pull request by mayur-satav**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:

- Before: https://main--pricefx-eds--pricefx.hlx.live/
- After: https://feature-pai-21-be-related-articles--pricefx-eds--pricefx.hlx.live/

### CODE REVIEW

File: component-definition.json [hunk 1]
```diff
@@ -331,6 +331,21 @@
               }
             }
           }
+        },
+        {
+          "title": "Related Article",
```
Comment (amogh-daryapurkar) [inline component-definition.json:336]: @mayur-satav Can we change this to Related Articles instead?

File: component-definition.json [hunk 2]
```diff
@@ -331,6 +331,21 @@
...
+          "id": "relatedArticle",
```
Comment (amogh-daryapurkar) [inline component-definition.json:337]: @mayur-satav Can we change this to Related Articles instead?

File: component-filters.json [hunk 3]
```diff
@@ -30,7 +30,8 @@
       "richtext",
       "spacer",
       "breadcrumb",
-      "socialshare"
+      "socialshare",
+      "relatedArticle"
```
Comment (amogh-daryapurkar) [inline component-filters.json:34]: @mayur-satav Can we change this to relatedArticles instead?

File: component-models.json [hunk 4]
```diff
@@ -792,5 +792,48 @@
         "valueType": "boolean"
       }
     ]
+  },
+  {
+    "id": "relatedArticle",
```
Comment (amogh-daryapurkar) [inline component-models.json:797]: @mayur-satav Can we change this to relatedArticles instead?

**Review by muneeskumar-rajendran — DISMISSED** [review]

**Review by amogh-daryapurkar — CHANGES_REQUESTED** [review]

**Review by amogh-daryapurkar — APPROVED** [review]

**Pull request merged**